### DISCUSSION TITLE
teensy 3.+ and 4.+ support

### DISCUSSION THREAD

**Discussion by raytong** [shanteacontrols/OpenDeck#49] [Q&A] [answered]
Hi,

Does OpenDeck support teensy 3.+ and 4.+ ?  thank you.

**Comment by paradajz** [comment, marked as answer]
No. Supported boards are clearly mentioned in the README.

**Comment by paradajz** [comment]
Teensy 4 support will be added over the next few days hopefully.

**Comment by paradajz** [comment]
Supported now.
